src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Select } from '../../types/Select';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  onSelect: (value: Select) => void;
+  activeTodos: Todo[];
+  select: Select;
+}
+
+export const Footer: React.FC<Props> = ({ onSelect, activeTodos, select }) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: select === Select.All },
+          )}
```

Comment:
Make it short & better
```suggestion
          className={cn('filter__link', {
             selected: select === Select.All 
           })}
```